samples/graph-change-notification-team-channel/csharp/ChangeNotification/Model/Notification.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+namespace ChangeNotification.Model
+{
+    using Newtonsoft.Json;
+    using System;
+
+    public class Notifications
+    {
+        [JsonProperty(PropertyName = "value")]
+        public Notification_List[] Items { get; set; }
+    }
+    public class Notification_List
```

Comment:
One line space before new class.
Also, don't user class name like this "Notification_List".
Rename it to some other meaningful name.

Reply:
fixed